kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/BrokerService.java

```diff
@@ -110,7 +111,7 @@ public Mono<Void> updateBrokerConfigByName(KafkaCluster cluster,
   }
 
   private Mono<Map<Integer, Map<String, DescribeLogDirsResponse.LogDirInfo>>> getClusterLogDirs(
-      KafkaCluster cluster, List<Integer> reqBrokers) {
+      KafkaCluster cluster, @Nullable List<Integer> reqBrokers) {
```

Comment:
let's use empty collections instead of nullable collections, less error-prone solution

Reply:
refacrored